sunpy/timeseries/sources/fermi_gbm.py

```diff
@@ -59,10 +59,53 @@ class GBMSummaryTimeSeries(GenericTimeSeries):
     # Class attribute used to specify the source class of the TimeSeries.
     _source = 'gbmsummary'
 
+    def plot(self, axes=None, title=None, **kwargs):
+        """
+        Plots the GBM lightcurve TimeSeries with data from its pandas dataframe.
+
+        Parameters
+        ----------
+        axes : `matplotlib.axes.Axes`, optional
+            The axes on which to plot the TimeSeries. Defaults to current axes.
+        title : `str`, optional
+            Plot title.
+        **kwargs : `dict`
+            Additional plot keyword arguments that are handed to `~matplotlib.axes.Axes.plot`
+            functions.
+
+        Returns
+        -------
+        `~matplotlib.axes.Axes`
+            The plot axes.
+        """
+
+        self._validate_data_for_plotting()
+
+        if axes is None:
+            axes = plt.gca()
+
+        data_lab = self.to_dataframe().columns.values
+
+        for d in data_lab:
+            axes.plot(self.to_dataframe().index, self.to_dataframe()[d], label=d, **kwargs)
+
+        axes.set_yscale("log")
+        if title is None:
+            axes.set_title('Fermi GBM Summary data ' + str(self.meta.get('DETNAM').values()))
+        else:
+            axes.set_title(title)
+        axes.set_xlabel('Start time: ' + self.to_dataframe().index[0].strftime('%Y-%m-%d %H:%M:%S UT'))
+        axes.set_ylabel('Counts/s/keV')
+        axes.legend()
+
+        return axes
+
     @peek_show
     def peek(self, **kwargs):
         """
-        Plots the GBM lightcurve TimeSeries. An example can be seen below:
+        Plots the GBM lightcurve TimeSeries by calling
+        `~sunpy.timeseries.sources.fermi_gbm.GBMSummaryTimeSeries.plot`.
+        An example can be seen below:
```

Comment:
```suggestion
```

Reply:
Every `peek()` method has this, do we want to get rid of this line?

Reply:
Sure.